### ISSUE TITLE
Cubase 12.0.60. 

### ISSUE THREAD

**Issue by danimusicos** [bjoluc/cubase-xtouch-midiremote#5]
Bjr, j'avais constaté un bug sur cubase 12.0.60 qui le figeait dès que je bougeais les pistes. J'ai donc mis à jour avec la version 1.5. C'est toujours pareil. J'ai essayé cubase 11, il n'y a pas de problème mais bien sûr, pas de prise en charge de contrôle midi. Ce qui pourrait expliquer le lien.

**Comment by bjoluc** [comment]
Hi @danimusicos,

thanks for reporting! Sadly, the issue you describe is caused by a [repoted issue](https://forums.steinberg.net/t/842187) introduced in Cubase 12.0.60. The workaround I released in v1.5 turned out to only work for some, but not all, users. I've been thinking about trying other workarounds too, but since I cannot reliably reproduce all the 12.0.60 issues, I decided against it. I'm hoping that Steinberg will follow up with a fix in the next months. Until then, we'll have to keep using Cubase 12.0.52 with the script...
Let's keep this issue open though until there's a solution to it.

Cheers!
